build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     ext {
         // build.version_qualifier parameter applies to knn plugin artifacts only. OpenSearch version must be set
         // explicitly as 'opensearch.version' property, for instance opensearch.version=2.0.0-rc1-SNAPSHOT
-        opensearch_version = System.getProperty("opensearch.version", "2.1.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
```

Comment:
I think opensearch version bump should be in a separate PR related to release, @junqiu-lei

Reply:
Sure, I can do that, part of https://github.com/opensearch-project/k-NN/issues/467 ?

Reply:
yes, I think so. I also like to get confirmation from @junqiu-lei, he's working on 2.2.0 release.

Reply:
Thanks @reta, I can take the version bump PR as part of 2.2 release.